apps/webapp/app/components/environments/EnvironmentLabel.tsx

```diff
@@ -32,7 +32,7 @@ export function environmentTitle(environment: Environment, username?: string) {
     case "STAGING":
       return "Staging";
     case "DEVELOPMENT":
-      return username ? `Dev: ${username}` : "Dev";
+      return username ? `Dev: ${username}` : "Dev: You";
```

Comment:
Wherever the Environment label is used it will be "Dev: You" now, or "Dev: username" for your team mates. This makes it crystal clear that there are multiple dev environments